### DISCUSSION TITLE
How do I do formatMessageCB for .czrc file?

### DISCUSSION THREAD

**Discussion by andy1uu** [Zhengqbbb/cz-git#201] [Q&A]
I want to use the custom formatting function for my .czrc file but I was not able to make it work, how would I convert the js function code to one that works with .czrc?

**Comment by Zhengqbbb** [comment]
All js fn config opts (*CB) not support on json 
1. Global: Create `~/.commitlintrc.js`
2. Project: 
    1. [Can also optionally custom configuration file path in package.json](https://cz-git.qbb.sh/config/#javascript-template)
    2. `czg --config=<file path>` - [Specify the configuration file to use](https://cz-git.qbb.sh/cli/config)

Btw: [FAQ: Configure path not as expected](https://cz-git.qbb.sh/faq/#configure-load-not-as-expected)

```
$ cat ~/.commitlintrc.js
module.exports = {
    prompt: {
        alias: {
            fd: 'docs: fix typos',
            f: 'docs: fix typos',
            a: 'feat(theme): update page theme',
            b: 'build: bump dependencies',
            c: 'chore: update config',
            s: 'style: update code format',
            ':': 'chore: update code and remove useless code',
        },
    },
}
```
